hokusai/commands/setup.py

```diff
@@ -119,6 +119,7 @@ def setup(aws_account_id, project_type, project_name, aws_ecr_region, port,
       if compose_environment == 'development':
         services[config.project_name]['command'] = development_command
         services[config.project_name]['ports'] = ["%s:%s" % (port, port)]
+        services[config.project_name]['volumes'] = ['../:/app']
```

Comment:
Yes - this means that changes made in the local project directory will be synced to the container's volume mounted at `/app` (where the code lives) - though one would still need to change the dev `command` to a hot-reloading dev server (i.e. shotgun) to see live changes reflected in their dev environment